src/components/templates/ImportResourceTemplate.jsx

```diff
@@ -131,11 +125,10 @@ class ImportResourceTemplate extends Component {
   render() {
     return (
       <div id="importResourceTemplate">
-        <UpdateResourceModal messages={this.props.modalMessages}
-                             update={this.handleUpdateResource} />
+        <UpdateResourceModal update={this.handleUpdateResource} />
         <Header triggerEditorMenu={this.props.triggerHandleOffsetMenu}/>
         <ImportFileZone setResourceTemplateCallback={this.setResourceTemplates} />
-        <CreateResourceMessages messages={this.state.flashMessages} />
+        <CreateResourceMessages />
```

Comment:
Should this be "CreateResourceTemplateMessages"?

Reply:
Possibly.  This wasn't changed or created in this PR though.